sheets/plugins.py

```diff
@@ -12,7 +12,7 @@ def refunds_process_request(
     ) -> RefundResult:
 
         refund_api_success = refund_order(
-            order_id=refund_request_row.order_id, unenroll=True
+            reference_number=refund_request_row.order_id, unenroll=True
```

Comment:
Needs to be fixed to use the new property:

```suggestion
            reference_number=refund_request_row.order_ref_num, unenroll=True
```